torch_geometric/loader/utils.py

```diff
@@ -24,6 +25,22 @@ def index_select(value: Tensor, index: Tensor, dim: int = 0) -> Tensor:
     return torch.index_select(value, 0, index, out=out)
 
 
+def unbatcher(src: torch.tensor, index: torch.tensor, dim: int = -1) -> list:
+    scatter = pandas.DataFrame(
+        index, columns=['scatter']).reset_index().groupby('scatter').tail(1)
```

Comment:
Any reason we need `pandas` for this? This should be doable via `torch-scatter` as well.

Reply:
We want to find where a graph end, then use ```index_select``` to select out the edges attributes of a graph. This also works for unbatching node_attrs